release_build_files/readme.md

```diff
@@ -569,6 +569,9 @@ code.
 ## Release Notes
 ### Next Release
 -   Changes
+    -   General (Android): Fixed a data race that could manifest as null pointer
```

Comment:
One last note (it can wait for the followup) - this isn't Android only, right? We can remove the "(Android)" tag.

Reply:
Ahh right. I've opened https://github.com/firebase/firebase-cpp-sdk/pull/749 to fix the release notes entry.